### PR TITLE
dbft: tune MaxBlock* parameters

### PR DESCRIPTION
**Pull request by roman-khimov**
Follow ExecFeeFactor changes for MaxBlockSystemFee and make it possible to fit
5K transactions into the block. Related to neo-project/neo-node#850.

### CODE REVIEW

File: src/DBFTPlugin/DBFTPlugin/config.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "IgnoreRecoveryLogs": false,
     "AutoStart": false,
     "Network": 860833102,
-    "MaxBlockSize": 262144,
-    "MaxBlockSystemFee": 900000000000
+    "MaxBlockSize": 2097152,
```
Comment (shargon) [inline src/DBFTPlugin/DBFTPlugin/config.json:7]: For more tx lower values?

Comment (roman-khimov) [reply src/DBFTPlugin/DBFTPlugin/config.json:7]: It's a bit different change, our public networks have substantially lowered ExecFeeFactor these days compared to the default, so MaxBlockSystemFee should be adjusted (9K GAS is _a lot_ of processing time). And it doesn't seem to be limiting, a typical NEO/GAS transfer transaction easily fits into 0.01 GAS of system fee. I don't have complete statstics for the whole chain lifetime, but recent Flamingo or non-native NEP-17 transactions seem to take 0.1-0.2 GAS in system fee, so 1K limit still seems to be reasonable.

Comment (shargon) [reply src/DBFTPlugin/DBFTPlugin/config.json:7]: These configuration should be related to the height 🤔

Comment (erikzhang) [reply src/DBFTPlugin/DBFTPlugin/config.json:7]: > These configuration should be related to the height

This is not necessary. These are just consensus node configurations and we can change them back at any time.

**Review by shargon — APPROVED** [review]

**Pull request merged**